cli/src/main.rs

```diff
@@ -14,9 +14,12 @@ const BUILD_VERSION: &'static str = env!("CARGO_PKG_VERSION");
 const BUILD_SHA: Option<&'static str> = option_env!("BUILD_SHA");
 
 fn main() {
-    if let Err(e) = run() {
+    if let Err(ref e) = run() {
```

Comment:
Is the `ref` needed here? I think the error can be moved into `e`, as opposed to borrowed.

Reply:
No, it isn't, I've removed it.